eng/Signing.props

```diff
@@ -1,16 +1,41 @@
 <Project>
 
   <ItemGroup>
-    <!-- Do not sign non-shipping packages -->
+    <!-- Do not sign non-shipping packages with doing in-build signing -->
     <ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)**\*.nupkg" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(PostBuildSign)' == 'true'">
```

Comment:
Are there any PowerShell scripts to sign, or are you only looking for containers that can hold other files?

Reply:
Those are added via the arcade defaults.